go/memcached-operator/bundle/manifests/cache.example.com_memcacheds.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@camilamacedo86 - As per https://github.com/operator-framework/operator-lifecycle-manager/pull/1416, the CRD defaults to v1 version than v1beta1 and k8s 1.19 will deprecate v1beta1 for CRD as such.. What about defaulting to v1 version of CRD in new layout?

Reply:
The problem is that when we run make bundle it is not copying the generated CRD's manifest and is re-creating it with the `v1beta1`. So, it shows a bug. I am raising an issue for that.

Reply:
Tks for raise your concern see: https://github.com/operator-framework/operator-sdk/issues/3580